packages/ng/grid/grid.component.ts

```diff
@@ -0,0 +1,64 @@
+import { NgTemplateOutlet } from '@angular/common';
+import { booleanAttribute, Component, computed, input, numberAttribute, ViewEncapsulation } from '@angular/core';
+
+@Component({
+	selector: 'lu-grid',
+	standalone: true,
+	templateUrl: './grid.component.html',
+	styleUrls: ['./grid.component.scss'],
+	encapsulation: ViewEncapsulation.None,
+	imports: [NgTemplateOutlet],
+})
+export class GridComponent {
+	container = input(false, { transform: booleanAttribute });
+	columns = input(null, { transform: numberAttribute });
+
+	mode = input<
```

Comment:
We could probably compose this type using the fancy types that we have made for columns